src/GeneralStateTestsFiller/stExtCodeHash/extCodeHashChangedAccountFiller.json

```diff
@@ -0,0 +1,89 @@
+{
+  "extCodeHashChangedAccount": {
+    "_info": {
+      "comment": "EXTCODEHASH/EXTCODESIZE of an account created then deleted in same transaction"
```

Comment:
the account changes its nonce and storage.

Reply:
I knew the copy/pasting would bite me eventually.  Fixed.